src/_langs/en/updates/2015-07-22-access-ble-devices-with-web-bluetooth.markdown

```diff
@@ -0,0 +1,279 @@
+---
+
+layout: update
+published: true
+
+collection: updates
+category: chrome
+product: chrome
+type: news
+date: 2015-07-22
+
+title: "Access BLE devices with Web Bluetooth"
+description: "A Web API has been added to Chrome OS M45 that
+makes it possible for websites to discover and communicate with devices over
+the Bluetooth 4 wireless standard using GATT."
+article:
+  written_on: 2015-07-22
+  updated_on: 2015-07-22
+
+authors:
+  - beaufortfrancois
+tags:
+  - Bluetooth
+  - IoT
+  - WebBluetooth
+  - PhysicalWeb
+permalink: /updates/2015/07/access-ble-devices-with-web-bluetooth.html
+---
+
+How about I tell you websites could communicate with nearby Bluetooth devices
```

Comment:
Perhaps: "What if I told you ..."
There's something slightly off with this phrasing.